--- wiredtiger-1.6.3/build_posix/wiredtiger.pc.in.orig	2026-05-17 19:13:41.981373736 +0200
+++ wiredtiger-1.6.3/build_posix/wiredtiger.pc.in	2026-05-17 19:13:54.634638520 +0200
@@ -7,5 +7,5 @@ Name: WiredTiger
 Description: The WiredTiger Data Engine
 Requires:
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lwiredtiger-@VERSION_NOPATCH@
-Cflags: -I${includedir}/wiredtiger-@VERSION_NOPATCH@
+Libs: -L${libdir} -lwiredtiger
+Cflags: -I${includedir}
