--- wiredtiger-11.3.1/cmake/configs/modes.cmake.orig	2024-11-14 23:24:41.000000000 +0100
+++ wiredtiger-11.3.1/cmake/configs/modes.cmake	2026-05-10 21:56:34.342682002 +0200
@@ -5,7 +5,7 @@ include(CheckCXXCompilerFlag)
 include(${CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
 
 # Establish an internal cache variable to track our custom build modes.
-set(BUILD_MODES Debug Release RelWithDebInfo CACHE INTERNAL "")
+set(BUILD_MODES Debug Release RelWithDebInfo PLD CACHE INTERNAL "")
 
 if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
     set(MSVC_C_COMPILER 1)
