--- wiredtiger-1.6.3/lang/python/Makefile.am.orig	2026-05-10 07:45:02.597144799 +0200
+++ wiredtiger-1.6.3/lang/python/Makefile.am	2026-05-10 08:38:29.659770650 +0200
@@ -13,7 +13,7 @@ _wiredtiger.so: $(top_builddir)/libwired
 	$(PYTHON) $(PYSRC)/setup.py build_ext -b . -t . -f $(PY_SETUP_DEBUG)
 
 install-exec-local:
-	$(PYTHON) $(PYSRC)/setup.py install_lib -b . --skip-build
+	$(PYTHON) $(PYSRC)/setup.py build -b . install --skip-build --root=$(DESTDIR) --prefix=/usr
 
 clean-local:
 	$(PYTHON) $(PYSRC)/setup.py clean
